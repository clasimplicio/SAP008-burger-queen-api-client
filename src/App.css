* {
    margin: 0 auto;
    padding: 0;
}
:root {
    --color-white: #FFFFFF;
    --color-black: #000000;
    --color-red: #B80303;
    --color-green: #1FB741;
    --color-gray: #9D9B9B;
    --color-background: #231F20;
    --color-gray-box: #403B3C;
    --color-yellow-service: #EBCE39;
    --color-yellow-kitchen: #F5E487;
}
body {
    background-color: var(--color-background);
    max-width: 810px;
}

.App {
    display: flex;
    flex-direction: column;
}

img {
    border: red solid 1px;
    margin-bottom: 80px;
    max-height: 50vh;
}

form {
    border: green solid 1px;
    background: cover no-repeat;
    display: flex;
    flex-direction: column;
    justify-content: center;
    row-gap: 2.5vh;
    width: 100vw;
    height: 50vh;
}

form > input {
    width: 60vw;
    height: 4.5vh;
    border-radius: 5px;
}

form > input[type=submit] {
    background-color: var(--color-yellow-service);
}
