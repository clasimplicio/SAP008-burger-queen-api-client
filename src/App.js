import burgerLogin from "./assets/burger.jpg"
import background from "./assets/background-login.svg"
import "../src/App.css"

function App() {
  return (
    <div className="App">
      <img src={burgerLogin} alt="logo-hamburger" />
      <form style={{backgroundImage: `url(${background})` }}>
        <input type="text" placeholder="USUÁRIO" />
        <input type="password" placeholder="SENHA" />
        <input type="submit" value="ENTRAR" />
      </form>
    </div>
  );
}

export default App;
