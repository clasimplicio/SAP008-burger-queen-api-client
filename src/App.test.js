import React from 'react'
import {render} from '@testing-library/react'
import App from './App'
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<App />)
    
    expect(asFragment(<App />)).toMatchSnapshot()
   })


// import '@testing-library/jest-dom';
// import userEvent from '@testing-library/user-event';
// import { render, screen } from '@testing-library/react';
// import { BrowserRouter } from 'react-router-dom';
// import React from 'react';
// import App from './App';

// describe('full app rendering/navigating', () => {   
    // render(<App />);
    // const linkElementLogin = screen.getByText(/Não possui conta/i);
    // const linkElementRegister = screen.getByText(/Já possui conta/i);

    // test('should go to SignUp', () => {
    //     expect(linkElementLogin).toBeInTheDocument();
    //     fireEvent.click(screen.getByText('Cadastre-se'));
    //     expect(screen.getByText(/Já possui conta/i)).toBeInTheDocument();
    // })

    // test('should go to Register', () => {
    //     expect(linkElementRegister).toBeInTheDocument();
    //     fireEvent.click(screen.getByText('Já possui conta'));
    //     expect(screen.getByText(/Cadastre-se/i)).toBeInTheDocument();
    // })

// });
