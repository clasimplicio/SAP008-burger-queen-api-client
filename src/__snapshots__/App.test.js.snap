// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <figure
    id="logo-login"
  >
    <img
      alt="logo-hamburger"
      id="logo-img-login"
      src="burger.png"
    />
  </figure>
  <form
    id="form-login"
  >
    <input
      class="inputForm"
      placeholder="E-MAIL"
      type="email"
    />
    <input
      class="inputForm"
      placeholder="SENHA"
      type="password"
    />
    <button
      class="btn_signin"
      type="button"
    >
      ENTRAR
    </button>
    <p
      id="errorMsg"
    />
    <p
      id="text-home"
    >
      NÃ£o possui conta? 
      <a
        class="link-home"
        href="/register"
      >
        Cadastre-se
      </a>
    </p>
  </form>
</DocumentFragment>
`;
